rosidl_cmake/package.xml

```diff
@@ -12,6 +12,7 @@
 
   <buildtool_export_depend>ament_cmake</buildtool_export_depend>
 
+  <exec_depend>python3-empy</exec_depend>
```

Comment:
Please see [REP 140](http://www.ros.org/reps/rep-0140.html) which describes the different dependency types. I think `buildtool_export_depend` makes more sense since in a cross-compile scenario `em` must exist for the host platform (not the target platform) since it is being invoked as part of building downstream packages.